pkg/utils/logs/cluster_logs.go

```diff
@@ -237,9 +238,25 @@ func (csr *ClusterStreamingRequest) streamInGoroutine(
 		}
 	}()
 
-	_, err = io.Copy(output, logStream)
-	if err != nil {
-		log.Printf("error sending logs to writer, pod %s: %v", podName, err)
-		return
+	scanner := bufio.NewScanner(logStream)
```

Comment:
We should increase the maximum line buffer as we do in all the other places.
```suggestion
	scanner := bufio.NewScanner(logStream)
	scanner.Buffer(make([]byte, 0, 4096), 1024*1024)
```